examples/plot_loco_minimal.py

```diff
@@ -0,0 +1,143 @@
+"""
```

Comment:
I don't like this naming. Why not `plot_loco.py` ?
The example is not minimal bzw ;-)